packages/ui/src/components/JobCard/Details/DetailsContent/JobLogs/JobLogs.tsx

```diff
@@ -127,6 +142,13 @@ export const JobLogs = ({ actions, job }: JobLogsProps) => {
                 data-line-number={`${log.lineNumber}.`}
               >
                 {log.message}
+                <Button
+                  onClick={copyLogToClipboard(log)}
```

Comment:
you've missed a function here, right?

Reply:
Oh, yeah seems so.

Strange that it still worked for me.

Reply:
because it saved to clipboard on each render